spec/integration/v2/order_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+require "flex_commerce_api"
+require "uri"
+
+RSpec.describe FlexCommerce::V2::Order do
+  # Global context for all specs - defines things you dont see defined in here
+  # such as flex_root_url, api_root, default_headers and page_size
+  # see api_globals.rb in spec/support for the source code
+  include_context "global context", api_version: "v2"
+  # As a number of specs are testing class methods, subject changes in different contexts
+  # so the subject class is defined here for convenience
+  let(:subject_class) { FlexCommerce::V2::Order }
+
+  #
+  # Member Examples
+  #
+  context "using a single resource" do
```

Comment:
Possibly worth separating the blocks inside here for readability (and maintain stylistic consistency)